test/rand_tangent.jl

```diff
@@ -11,6 +11,8 @@ using FiniteDifferences: rand_tangent
         (:a, NoTangent),
         (true, NoTangent),
         (4, NoTangent),
+        (FiniteDifferences, NoTangent),
+        (Foo, NoTangent),
```

Comment:
Command saying waht it is, and mayube also we should check different kinds of types? (might be overkill though?
```suggestion
        (FiniteDifferences, NoTangent),  # Module object
        # Types (not instances of type)
        (Foo, NoTangent),  
        (Union{Int, Foo}, NoTangent),
        (Union{Int, Foo}, NoTangent),
        (Vector, NoTangent),
        (Vector{Float64}, NoTangent),
        (::Integer, NoTangent),
        (Type{<:Real}, NoTangent),
```